ios/KochavaTracker.h

```diff
@@ -0,0 +1,27 @@
+//
+//  KochavaTracker.h
+//  KochavaTracker
+//
+//  Created by John Bushnell on 4/16/19.
+//  Copyright © 2013 - 2020 Kochava, Inc.  All rights reserved.
+//
+
+
+
+#ifndef KochavaTracker_h
+#define KochavaTracker_h
+
+
+
+#import "KochavaCore.h"
```

Comment:
@rampr 
Here `#import "KochavaCore.h"` needs to imported which was actually not present in the tracker sdk which I downloaded, it's present in core sdk. So just copy pasting the `KochavaCore.h` won't work since it has dependencies which are from code sdk. So I had to import tracker sdk and core sdk since kochava became a modular. 
cc: @hatim-developer